src/Nethermind/Nethermind.Runner/configs/hoodi.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "https://raw.githubusercontent.com/NethermindEth/core-scripts/refs/heads/main/schemas/config.json",
+  "Init": {
+    "ChainSpecPath": "chainspec/hoodi.json",
+    "GenesisHash": "0xbbe312868b376a3001692a646dd2d7d1e4406380dfd86b98aa8a34d1557c971b",
+    "BaseDbPath": "nethermind_db/hoodi",
+    "LogFileName": "hoodi.log"
```

Comment:
Usualy we have convention like:
chiado.logs.txt -> Can we adjust to be sure that we keep it in same way everywhere?